src/components/containers/Characters.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import DeckOfCards from '../DeckOfCards';
+import PropTypes from 'prop-types';
+import apiCall from '../../services/api-call';
+
+class Characters extends Component {
+  static propTypes = {
+    charCount: PropTypes.number.isRequired
+  }
+  state= {
+    characters: [],
+    pageCount: 1
+  }
+
+  componentDidMount() {
+    apiCall('https://hey-arnold-api.herokuapp.com/api/v1/characters', this.props.charCount, this.state.pageCount)
```

Comment:
i would set this url as a const outside your class so you dont have to write it several times. helps with typos